src/controllers/projectTaskController.js

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
```

Comment:
better to separate file imports according to their types
```suggestion
import { Router } from 'express';
 import HttpStatus from 'http-status-codes';
 
 import logger from '../utils/logger';
 
 import * as projectTaskService from '../services/projectTaskService';
```

Reply:
🆗